pydicom/valuerep.py

```diff
@@ -27,6 +27,11 @@
 # and PN, but it is handled separately.
 text_VRs = ('SH', 'LO', 'ST', 'LT', 'UC', 'UT')
 
+# Delimiters for text strings and person name that reset the encoding.
+# See PS3.5, Section 6.1.2.5.3
+TEXT_VR_DELIMS = (b'\n', b'\r', b'\t', b'\f')
+PN_DELIMS = (b'^', )
+
```

Comment:
Would it be worth handling the = delimiter in the same manner?

Reply:
I left this out because of multi-value encodings, which should be split by '=', but not by the other delimiters - actually I want to handle multi-value encodings the same way, but I couldn't get it to work correctly yet. Will check this later...